.travis.yml

```diff
@@ -11,3 +11,4 @@ before_install:
   - sudo apt-get update -qq
   - sudo apt-get install libzmq3
   - sudo apt-get install libzmq3-dev
+  - gem update bundler
```

Comment:
btw since you're touching that file you might want to update the ruby versions :memo: